receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -94,6 +98,15 @@ func (prw *prometheusRemoteWriteReceiver) Start(ctx context.Context, host compon
 	mux.HandleFunc("/api/v1/write", prw.handlePRW)
 	var err error
 
+	prw.obsrecv, err = receiverhelper.NewObsReport(receiverhelper.ObsReportSettings{
+		ReceiverID:             prw.settings.ID,
+		Transport:              "http",
+		ReceiverCreateSettings: prw.settings,
+	})
+	if err != nil {
+		return fmt.Errorf("failed to create obsreport: %w", err)
+	}
+
```

Comment:
This change sounds unrelated to target_info handling, could we do it in a separate PR?

Reply:
[7739e72](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38812/commits/7739e726e3c6869ac2eb6238edaf248429417f7b)

Done here. I know that you mentioned to not put a TODO task here, but I can't rely on my memory 🫠 to remeber it

Reply:
fair enough :)